images/base/Dockerfile

```diff
@@ -118,7 +118,7 @@ RUN eval "$(gimme "${GO_VERSION}")" \
 # stage for building containerd
 FROM go-build as build-containerd
 ARG TARGETARCH GO_VERSION
-ARG CONTAINERD_VERSION="v1.7.5"
+ARG CONTAINERD_VERSION="v1.7.13"
```

Comment:
We don't depend on containerd for runc binary version (Though it's recommended to keep in sync), was debating earlier if we should just bump runc.

Reply:
probably best to just run through relnotes for both at this point, even if it takes a little longer